src/common/ContractInteractor.ts

```diff
@@ -69,8 +69,8 @@ export const StakeUnlocked: EventName = 'StakeUnlocked'
 export const StakeWithdrawn: EventName = 'StakeWithdrawn'
 export const StakePenalized: EventName = 'StakePenalized'
 
-export const WAIT_FOR_RECEIPT_RETRIES = 5;
-export const WAIT_FOR_RECEIPT_INITIAL_BACKOFF = 1000;
+export const WAIT_FOR_RECEIPT_RETRIES = 5
```

Comment:
This instead of constants can be parameters sent on the relay client constructor or even in the transaction so you don't have hardcoded constants here.